src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/uploadapprovedorder/UploadApprovedOrderContestedAboutToStartHandlerTest.java

```diff
@@ -22,7 +22,7 @@
 import static uk.gov.hmcts.reform.finrem.caseorchestration.TestConstants.AUTH_TOKEN;
 
 @RunWith(MockitoJUnitRunner.class)
-public class UploadApprovedOrderContestedAboutToStartHandlerTest extends UploadApprovedOrderBaseHandlerTest {
+public class UploadApprovedOrderContestedAboutToStartHandlerTest extends UploadApprovedOrderBaseHandlerSetup {
```

Comment:
Same here, I think having a Parent class following the same name pattern is less confusing. preference again...

Reply:
renamed - kept Test in Class name